pages/dashboard/dashboard.css

```diff
@@ -0,0 +1,50 @@
+.cards {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(18.75rem, 30%));
+  grid-gap: 1.25rem;
+  margin-top: 3.75rem;
+}
+
+.card {
+  background-color: var(--white);
+  border-radius: 5px;
+  padding: 0.625rem;
+  box-shadow: 0 0 10px #0000001a;
+  cursor: pointer;
+}
+
+img {
+  width: 100%;
+  height: 200px;
+  object-fit: cover;
+  border-radius: 5px;
+}
+
+h2 {
+  padding: 1.25rem 0;
+}
+
+.tags {
+  display: flex;
```

Comment:
mislim da ti ne treba ovdje flex, probaj bez toga, sto mislis?

Reply:
Ja bih to pustila jer ako budemo imali više tagova da tagovi idu u novi red.
![image](https://github.com/tonkec/recepti_alumni/assets/81707177/d559caeb-6118-4486-9b67-870f77619f3e)